packages/fxa-auth-server/lib/senders/emails/partials/paymentProvider/en.ftl

```diff
@@ -0,0 +1,6 @@
+payment-method = Payment method: PayPal
```

Comment:
I think we're going to want a `span` and l10n ID specifically for "Payment method:". That way we can add "PayPal" as a placeholder (probably `{ -paypal }` the same way we do for other brands) and also use it for "Payment method: Mastercard ending in 1234" for example for cc cards, which I think is what Jody was suggesting with his comment.